### ISSUE TITLE
[Bug]: Upgrade fails when upgrading from 1.6.16 --> 1.6.17 using Docker/Watchtower

### ISSUE THREAD

**Issue by andrewtlove** [appsmithorg/appsmith#12695]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Appsmith is deployed to DigitalOcean using the DO marketplace function (Docker) and using custom domain.  When watchtower attempts to apply upgrade, the app fails to come back online, throwing the following error:

```bash
2022-04-07 19:01:09,982 INFO success: editor entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
editor stdout | Load environment configuration
editor stdout | 
editor stderr | + http_conf=/opt/appsmith/templates/nginx-app-http.conf.template.sh
editor stderr | + https_conf=/opt/appsmith/templates/nginx-app-https.conf.template.sh
editor stderr | + ssl_conf_path=/appsmith-stacks/data/certificate/conf
editor stderr | + APP_TEMPLATE=/opt/appsmith/templates/nginx-app-http.conf.template.sh
editor stderr | + mkdir -pv /appsmith-stacks/data/certificate/conf
editor stderr | 
editor stderr | + cat
editor stderr | 
editor stderr | + cat
editor stderr | 
editor stderr | + [[ -n sub.domain.tld ]]
editor stderr | /opt/appsmith/run-nginx.sh: line 46: DYNO: unbound variable
editor stderr | 
2022-04-07 19:01:09,995 INFO exited: editor (exit status 1; not expected)
```

Application never recovers.  In order to fix it, I had to roll back using a backup and manually disable watchtower.

### Steps To Reproduce

1. Set up appsmith v1.6.17 using Digital Ocean
2. Wait for watchtower cron to run
3. ???
4. Fail

### Version

Self Hosted v 1.6.17

**Comment by AS-Laguna** [comment]
I believe this has been fixed with [this commit](https://github.com/appsmithorg/appsmith/commit/7807d5f5c695003dbcfb0c6f18b09810e698f44d). @mohanarpit can you confirm?

**Comment by mohanarpit** [comment]
This has already been fixed with v1.6.18. Would request you to upgrade to this version. 

Apologies for the poor experience here.
